chemprop/nn/predictors.py

```diff
@@ -203,6 +204,27 @@ def forward(self, Z: Tensor) -> Tensor:
     train_step = forward
 
 
+@PredictorRegistry.register("regression-quantile")
+class QuantileFFN(RegressionFFN):
+    n_targets = 2
+    _T_default_criterion = QuantileLoss
+
+    def forward(self, Z: Tensor) -> Tensor:
+        Y = super().forward(Z)
+        lower_bound, upper_bound = torch.chunk(Y, self.n_targets, 1)
```

Comment:
I though uncertainty targets were in the final dimension now?

Reply:
The quantile regression will predict the $\alpha$ and $1 - \alpha$ quantiles for a given input. These two quantiles are then used to calculate the mean as the prediction and the interval as the uncertainty. The forward function in `QuantileFFN` will still return the uncertainty targets in the final dimension (see L222).

Reply:
oh yes my bad- thanks for the clarification.